views/account.pug

```diff
@@ -45,21 +45,21 @@ block content
                 if trace.type === "reward" && trace.action.author === account.address
                   tr
                     td
-                      a(href="/block/" + trace.blockNumber) #{trace.blockHash.substr(0, 10)}...
+                      a(href="/explorer/block/" + trace.blockNumber) #{trace.blockHash.substr(0, 10)}...

```

Comment:
Instead of messing with every singel route I'm sure there must be a way to set a prefix in express?

Reply:
I think we need each service on its own subdomain and/or IP address.

Reply:
i tried the express thing, it didn't work.